src/pudl/transform/ferc1.py

```diff
@@ -2925,7 +2925,10 @@ def transform(
 
 
 def ferc1_transform_asset_factory(
-    table_name: str, ferc1_tfr_classes: dict[str, Ferc1AbstractTableTransformer]
+    table_name: str,
+    ferc1_tfr_classes: Mapping[str, type[Ferc1AbstractTableTransformer]],
+    io_manager_key: str = "pudl_sqlite_io_manager",
+    convert_dtypes: bool = True,
```

Comment:
I had to add some args here to accommodate for the differences between the `Ferc1AbstractTableTransformer` and the `GenericPlantFerc1TableTransformer`.